### ISSUE TITLE
Implement NaN values in SMA Messages

### ISSUE THREAD

**Issue by samsta** [samsta/BatteryController#1]
Messages need to be able to convey that some values are invalid, e.g. when we haven't received that information from the Battery (yet) or when the simply don't apply (e.g. missing sensor).

That's what the column AC in `Value List -CAN - Modified.xlsx` on dropbox is for.

**Comment by NiallDarwin** [comment]
I don't think we care about SMA anymore.. Close this?

**Comment by samsta** [comment]
Yep - do you want to delete everything SMA?

**Comment by NiallDarwin** [comment]
Can it be deleted in a reversible way - ie if we ever end up going back to the stanists we can resurrect it?

**Comment by samsta** [comment]
That’s the whole point of using a version control system like git. It remembers the history back to the dawn of time and any previous version can be resurrected

**Comment by NiallDarwin** [comment]
Sometimes the thing that's the whole point of a technology ends up not working. EG cars and roads for rapid transport and freedom leads to traffic jams and people enslaved to paying for their vehicles.
However I take it you're saying git actually works and it won't be an absolute git to resurrect the SMA stuff if needed. Getting it to integrate with however the program has evolved may be a different story but that's life.
In short, yes, let's kill all the SMA stuff.
